ci/bazel-scripts/main.sh

```diff
@@ -23,9 +23,11 @@ fi
 # - target branch is not rc--*
 # - uploading to S3 has not been requested
 
-if [[ "${CI_MERGE_REQUEST_TITLE:-}" == *"[RUN_ALL_BAZEL_TARGETS]"* ]] || [[ "${CI_MERGE_REQUEST_TITLE:-}" == *"[S3_UPLOAD]"* ]]; then
+if [[ "${CI_MERGE_REQUEST_TITLE:-}" == *"[RUN_ALL_BAZEL_TARGETS]"* ]]; then
```

Comment:
This is the behaviour we had previously but we with that we got confused users because even though they had added `[S3_UPLOAD]` to their PR titles they still couldn't find their artefacts. This is why we made `[S3_UPLOAD]` behave the same as `[RUN_ALL_BAZEL_TARGETS]`.

Reply:
That was added in: https://github.com/dfinity/ic/commit/73b4d99c512ec71a7d33b3e92f0f9ce885f5ebc7

Reply:
In that case I would suggest we get rid of `[S3_UPLOAD]` all together. Could we just have one label that is `BUILD_ALL_TARGETS_AND_UPLAOD`? I can add a deprecation message for the old flags.

Reply:
I'm fine with that as well. Do also adapt `gitlab-ci/src/ci-scripts/build-ic.sh` similarly.

Reply:
It can just stay `RUN_ALL_TARGETS` but with uploading artifacts.